core/loadpoint_plan.go

```diff
@@ -169,10 +169,10 @@ func (lp *Loadpoint) plannerActive() (active bool) {
 			// don't stop an already running slot if goal was not met
 			lp.log.DEBUG.Println("plan: continuing until end of slot")
 			return true
-		case requiredDuration < smallGapDuration:
+		case requiredDuration < smallSlotDuration:
 			lp.log.DEBUG.Printf("plan: continuing for remaining %v", requiredDuration.Round(time.Second))
 			return true
-		case lp.clock.Until(planStart) < smallGapDuration:
+		case requiredDuration > smallSlotDuration && lp.clock.Until(planStart) < smallGapDuration:
```

Comment:
Hier müsste doch ein required < smallSlot hin damit das Sinn ergibt?

Reply:
In der Zeile drüber haben wir diese Prüfung schon. 
https://github.com/evcc-io/evcc/pull/16815/files#diff-fe45e1679a0222b08663aa4f9090815d511e1145e92dc74b068326818a989235R172

Ich würde sagen hier kommt man hin wenn man zb. noch 20 min zum laden hat und das nächste segment in unter 30min anfangen würde, dann würde man einfach weiterladen. 
Wenn man zb. noch 40min übrig hat und das nächste segment in über 30 min los geht dann würde pausiert werden.

Reply:
Du hast recht. Die zusätzliche Bedingung kann aus dem Grund aber weg- wenn man zu dem Case kommt ist sie schon gegeben.